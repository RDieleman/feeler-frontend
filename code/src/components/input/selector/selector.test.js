import React from "react";
import {shallow} from "enzyme";
import SelectorComponent from "./selector.component";

it("Expect to render Selector component", () => {
    const children = ["test1", "test2", "test3"];
    expect(shallow(
        <SelectorComponent
            options={children}
        />)).toMatchSnapshot();
});