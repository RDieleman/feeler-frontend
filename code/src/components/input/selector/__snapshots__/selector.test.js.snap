// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Selector component 1`] = `ShallowWrapper {}`;
